### ISSUE TITLE
useQuantitativeDescriptions name violates PhET naming convention 

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/molarity#101]
From the [code-review checklist](https://github.com/phetsims/phet-info/blob/master/checklists/code_review_checklist.md):

> - [ ] Naming for Property values: All AXON/Property instances should be declared with the suffix Property. For example, if a visible property is added, it should have the name visibleProperty instead of simply visible. This will help to avoid confusion with non-Property definitions.

In MolarityScreenView:

```js
    // Determines whether qualitative or quantitative a11y descriptions are used.
    const useQuantitativeDescriptions = new DerivedProperty( [ valuesVisibleProperty ], () => {
      return valuesVisibleProperty.value;
    } );
```

This should be named `useQuantitativeDescriptionsProperty`.  By convention, a Property is always named with a "Property" suffix.  

This name also needs to be fixed in MolarityScreenSummaryNode and *Describer classes.

**Comment by pixelzoom** [comment]
This name also needs to be fixed in MolarityScreenSummaryNode and all *Describer classes.

**Comment by twant** [comment]
Thanks for catching this, @pixelzoom. Updated and assigned back over to you.

**Comment by pixelzoom** [comment]
Perfect @twant, thanks for making the change.  Closing.
